File: app/src/main/java/vn/loitp/app/activity/demo/ebookwithrealm/adapters/RealmBooksAdapter.java
package vn.loitp.app.activity.demo.ebookwithrealm.adapters;

import android.content.Context;

import io.realm.RealmResults;
import vn.loitp.app.activity.demo.ebookwithrealm.model.Book;

public class RealmBooksAdapter extends RealmModelAdapter<Book> {

    public RealmBooksAdapter(Context context, RealmResults<Book> realmResults, boolean automaticUpdate) {

        super(context, realmResults, automaticUpdate);
    }
}

File: app/src/main/java/vn/loitp/app/activity/demo/ebookWithRealm/adapters/RealmBooksAdapter.kt
package vn.loitp.app.activity.demo.ebookWithRealm.adapters

import android.content.Context
import io.realm.RealmResults
import vn.loitp.app.activity.demo.ebookWithRealm.model.Book

class RealmBooksAdapter(
    context: Context,
    realmResults: RealmResults<Book>,
    automaticUpdate: Boolean
) : RealmModelAdapter<Book>(
    context,
    realmResults,
    automaticUpdate
)
